autosklearn/evaluation/train_evaluator.py

```diff
@@ -28,7 +28,7 @@
                                    'LeavePGroupsOut': {'n_groups': 2},
                                    'LeaveOneOut': {},
                                    'LeavePOut': {'p': 2},
-                                   'PredefinedSplit': {},
+                                   'PredefinedSplit': {'test_fold': [0, 1, 2, 3]},
```

Comment:
Could this also be an empty list?

Reply:
PredefinedSplit with empty list will return no index(None) after split, so I do not think it will work.